src/utils/bat/output.rs

```diff
@@ -13,6 +13,28 @@ use crate::env::DeltaEnv;
 use crate::fatal;
 use crate::features::navigate;
 
+#[derive(Debug, Default)]
+pub struct PagerCfg {
+    pub navigate: bool,
+    pub show_themes: bool,
```

Comment:
I should have added a comment on this code...

```rust
                if config.show_themes {
                    p.arg("+n");
                }
```                

If you happen to know or discover what the logic is do feel free to add a comment!

Reply:
`+cmd`, "specified cmd to be executed", so on start is it as if the user pressed 'n' to continue the search.

Reply:
Right thanks, I'll add that as a comment.